variables.tf

```diff
@@ -86,3 +86,13 @@ variable "is_organization_trail" {
   default     = "false"
   description = "The trail is an AWS Organizations trail"
 }
+
+variable "enable_sns" {
+  default     = "false"
+  description = "If true, enable sns_topic_name with value"
+}
```

Comment:
```suggestion
```
let's get rid of `enable_sns`

Reply:
OK, wasn't sure, seemed like the enable was more in line with the rest of the module's "UI".